### PR TITLE
changes

### PR DESCRIPTION
**Pull request by rigouz**
- so many changes
- so much improvement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,4 @@ And *again and again.*
 This can also be edited in the web UI! **Instant** preview!
 
 Everything is nicer on a branch.
-Flowers are everywhere!!!
-
-And bees.
-
-And the sky is blue.
+Flowers are everywhere!!! And bees. And markdown.
```
Comment (walakj) [inline README.md:10]: seriously, markdown?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# owlcode
+# owlcode, the code of owls.
 
 Read me carefully.
 Read me again.
```
Comment (walakj) [inline README.md]: not again!

**Review by walakj — CHANGES_REQUESTED** [review]
address my feedback, buddy

File: README.md (hunk 1 above)
Comment (rigouz) [reply README.md:10]: yes, markdown! What did you expect?

File: README.md (hunk 2 above)
Comment (rigouz) [reply README.md]: Again and again.

**Comment by walakj** [comment]
I seriously need some changes. Or else.

**Comment by rigouz** [comment]
Happy now?

**Review by walakj — APPROVED** [review]

**Pull request merged**